app/models/event.rb

```diff
@@ -51,6 +51,11 @@
 #
 class Event < ApplicationRecord
   MIN_WAITING_TIME_BETWEEN_FEES = 5.days
+  BREAKDOWN_TIMEFRAMES = {
+    "Past year"  => 1.year,
+    "Past month" => 1.month,
+    "Past week"  => 1.week
+  }.freeze
```

Comment:
Can we move this to `BreakdownEngine::TIMEFRAMES`
- Create a file called `app/services/breakdown_engine.rb`